src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -0,0 +1,170 @@
+		{
+			if ($serversInResourceGroup[$i].Name -eq $serverName)
+			{
+				$found = 1
+				Assert-AreEqual $location $serversInResourceGroup[$i].Location
+				Assert-AreEqual "Microsoft.AnalysisServices/servers" $serversInResourceGroup[$i].Type
+				Assert-True {$serversInResourceGroup[$i].Id -like "*$resourceGroupName*"}
+
+				break
+			}
+		}
+		Assert-True {$found -eq 1} "server created earlier is not found when listing all in resource group: $resourceGroupName."
+
+		# List all Analysis Services servers in subscription
+		[array]$serversInSubscription = Get-AzureRmAnalysisServicesServer
+		Assert-True {$serversInSubscription.Count -ge 1}
+		Assert-True {$serversInSubscription.Count -ge $serversInResourceGroup.Count}
+    
+		$found = 0
+		for ($i = 0; $i -lt $serversInSubscription.Count; $i++)
+		{
+			if ($serversInSubscription[$i].Name -eq $serverName)
+			{
+				$found = 1
+				Assert-AreEqual $location $serversInSubscription[$i].Location
+				Assert-AreEqual "Microsoft.AnalysisServices/servers" $serversInSubscription[$i].Type
+				Assert-True {$serversInSubscription[$i].Id -like "*$resourceGroupName*"}
+    
+				break
+			}
+		}
+		Assert-True {$found -eq 1} "Account created earlier is not found when listing all in subscription."
+
+		# Suspend Analysis Servicesserver
+		Suspend-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName
+		[array]$serverGet = Get-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName
+		$serverGetItem = $serverGet[0]
+		Assert-True {$serverGetItem.ProvisioningState -like "Paused"}
+
+		# Resume Analysis Servicesserver
+		Resume-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName
+		[array]$serverGet = Get-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName
+		$serverGetItem = $serverGet[0]
+		Assert-True {$serverGetItem.ProvisioningState -like "Succeeded"}
+		
+		# Delete Analysis Servicesserver
+		Remove-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName -Force -PassThru
+
+		# Verify that it is gone by trying to get it again
+		Assert-Throws {Get-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName}
+	}
+	finally
+	{
+		# cleanup the resource group that was used in case it still exists. This is a best effort task, we ignore failures here.
+		Invoke-HandledCmdlet -Command {Remove-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName -Force -ErrorAction SilentlyContinue} -IgnoreFailures
+		Invoke-HandledCmdlet -Command {Remove-AzureRmResourceGroup -Name $resourceGroupName -Force -ErrorAction SilentlyContinue} -IgnoreFailures
+	}
+}
+
+
+<#
+.SYNOPSIS
+Tests Analysis Services server lifecycle  Failure scenarios (Create, Update, Get, Delete).
+#>
+function Test-NegativeAnalysisServicesServer
+{
+    param
+	(
+		$location = "West US",
```

Comment:
The tests have to be run in live mode when re-recorded.  Hard-coded locations are a concern for long-term maintainability of the tests.  Please file an issue for this and send it to us and you can fix it in the next sprint.